src/main/java/at/hannibal2/skyhanni/config/features/fishing/BarnTimerConfig.java

```diff
@@ -21,6 +21,14 @@ public class BarnTimerConfig {
     @FeatureToggle
     public Property<Boolean> enabled = Property.of(true);
 
+    @Expose
+    @ConfigOption(
+        name = "Show Anywhere",
+        desc = "Show the Barn Fishing Timer whenever you fish up a sea creature, regardless of location."
+    )
+    @ConfigEditorBoolean
+    public Property<Boolean> showAnywhere = Property.of(true);
```

Comment:
i dont think this should be on by default

Reply:
I left it as true by default due to the other entries being true by default as well. It's not an invasive option, so if a player doesn't like it, they can always disable it.